### ISSUE TITLE
Consider clearing the search field when starting the app

### ISSUE THREAD

**Issue by mihe** [Mosc/Glider#150]
This is arguably a subjective issue, and might even have been deliberately put in to begin with, but as of right now (2.5.0) the search field is never cleared under any circumstance as far as I can tell.

While this is a nice feature to have when you're maybe modifying the search slightly or just want to go back to the results, I've found it somewhat jarring when opening the search after a couple of days and find that my old search query is still there, forcing me to manually erase it before performing the new search.

I feel like this persistence of the search query is perhaps best served during the same app "session" and should perhaps be cleared when restarting the app.

I'm using Glider on a OnePlus 8T running Android 13, if that matters.

**Comment by mihe** [comment]
I guess a decent compromise could be for the search query to always be entirely selected when you first press the search button.

That way you can still modify the previous search if you want to, but if your intention is to perform a new search you can just start typing right away and it'll overwrite the previous query by default.

**Comment by Mosc** [comment]
Agreed. I've removed the persistence of search queries (across sessions) in 675c4e01c4fcc86e0286f586e142c99b02a6c624 and released it as part of v2.6.0. Honestly... I don't really know why that ever made sense in the first place. I experimented with your second suggestion of selecting the query when opening the search screen, but that behavior felt a bit off to me.
